lib/screens/home/widgets/category_main_menu.dart

```diff
@@ -15,33 +17,40 @@ class _CategoryMainMenuState extends ConsumerState<CategoryMainMenu> {
     final gridTile = ref
         .read(homeViewModelProvider.notifier)
         .cocktailMenuTiles
-        .map((tile) => _buildGridTile(withName: tile.name))
+        .map((tile) => _buildGridTile(withName: tile.name, path: tile.path))
         .toList();
 
     return Center(
         child: GridView.count(
       primary: false,
       padding: const EdgeInsets.all(20),
```

Comment:
Remember to add constant on this too, I know it was my fault, but it was just to serve as a filing example :P